src/types/Revurdering.ts

```diff
@@ -1,18 +1,15 @@
 import { Behandling } from './Behandling';
 import { Beregning } from './Beregning';
 
-export interface OpprettetRevurdering {
+export interface Revurdering {
     id: string;
     status: RevurderingsStatus;
     opprettet: string;
     tilRevurdering: Behandling;
 }
+export type OpprettetRevurdering = Revurdering & Record<string, never>;
```

Comment:
Hva er greia med denne?

Reply:
Det var för att införa en type med ett annat navn som var annars helt lik `Revurdering`.

Reply:
Hvorfor har den da `& Record<string, never>`?

Reply:
for mer stilpoäng hehe. kanske bara kunde varit `type x = Revurdering` ?